### ISSUE TITLE
NoMethodError - Can't run pod install

### ISSUE THREAD

**Issue by lenkaiser** [grab/cocoapods-pod-merge#4]
While trying to convert a few pods to a group I encountered the following error. At first I grouped Swift and ObjC pods together but then I read the tips section and split them. This didn't fix my issues though.

### MergeFile
```
group 'Obj_C'
    pod 'GCDWebServer'
    pod 'DTCoreText'
    pod 'PureLayout'
end
```

### Podfile
```
pod 'Obj_C', :path => 'MergedPods/Obj_C'
```

### Error
```
NoMethodError - undefined method `map' for "Core/Source/default.css":String
Did you mean?  tap
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:355:in `block in extract_info_from_podspec'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:354:in `each'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:354:in `extract_info_from_podspec'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:267:in `block (2 levels) in merge'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:266:in `chdir'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:266:in `block in merge'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:214:in `each'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:214:in `merge'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:68:in `block in begin'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:67:in `each'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods-pod-merge/Main.rb:67:in `begin'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-pod-merge-0.0.1/lib/cocoapods_plugin.rb:8:in `block in <module:CocoapodsPodMerge>'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/hooks_manager.rb:124:in `block (3 levels) in run'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/user_interface.rb:145:in `message'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/hooks_manager.rb:116:in `block (2 levels) in run'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/hooks_manager.rb:115:in `each'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/hooks_manager.rb:115:in `block in run'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/user_interface.rb:145:in `message'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/hooks_manager.rb:114:in `run'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:599:in `run_plugins_pre_install_hooks'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:219:in `block in prepare'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/user_interface.rb:145:in `message'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:215:in `prepare'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/installer.rb:155:in `install!'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/command/install.rb:52:in `run'
/Library/Ruby/Gems/2.6.0/gems/claide-1.0.3/lib/claide/command.rb:334:in `run'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/lib/cocoapods/command.rb:52:in `run'
/Library/Ruby/Gems/2.6.0/gems/cocoapods-1.8.4/bin/pod:55:in `<top (required)>'
/usr/local/bin/pod:23:in `load'
/usr/local/bin/pod:23:in `<top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli/exec.rb:74:in `load'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli/exec.rb:74:in `kernel_load'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli/exec.rb:28:in `run'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli.rb:463:in `exec'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli.rb:27:in `dispatch'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/cli.rb:18:in `start'
/Library/Ruby/Gems/2.6.0/gems/bundler-1.17.2/exe/bundle:30:in `block in <top (required)>'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/bundler/friendly_errors.rb:124:in `with_friendly_errors'
/Library/Ruby/Gems/2.6.0/gems/bundler-1.17.2/exe/bundle:22:in `<top (required)>'
/usr/bin/bundle:23:in `load'
/usr/bin/bundle:23:in `<main>'
```

**Comment by biocross** [comment]
@lenkaiser  Thanks a lot for reporting this!

I found the issue and just released version [0.0.2](https://github.com/grab/cocoapods-pod-merge/releases/tag/0.0.2) with the fix. 

Unfortunately, the `MergeFile` you mentioned will still not work at this time. This is because `DTCoreText` is a fairly complex pod, and contains `.c` files, which the plugin implicitly ignores while merging at at time. (You might have noticed this in the Tips as well)

On removing that pod, the other two pods merge just fine.
```
group 'Obj_C'
    pod 'GCDWebServer'
    pod 'PureLayout'
end
```

I can take a look at supporting more complex Pods in the future, but I cannot promise anything :)

**Comment by biocross** [comment]
Fixed in Release [0.0.2](https://github.com/grab/cocoapods-pod-merge/releases/tag/0.0.2)

**Comment by lenkaiser** [comment]
> @lenkaiser Thanks a lot for reporting this!
> 
> I found the issue and just released version [0.0.2](https://github.com/grab/cocoapods-pod-merge/releases/tag/0.0.2) with the fix.
> 
> Unfortunately, the `MergeFile` you mentioned will still not work at this time. This is because `DTCoreText` is a fairly complex pod, and contains `.c` files, which the plugin implicitly ignores while merging at at time. (You might have noticed this in the Tips as well)
> 
> On removing that pod, the other two pods merge just fine.
> 
> ```
> group 'Obj_C'
>     pod 'GCDWebServer'
>     pod 'PureLayout'
> end
> ```
> 
> I can take a look at supporting more complex Pods in the future, but I cannot promise anything :)

Of course, you're right about the C files. I read about this but I forgot to check due to the error that the plugin presented.
